src/chat.rs

```diff
@@ -1945,6 +1945,7 @@ async fn prepare_msg_blob(context: &Context, msg: &mut Message) -> Result<()> {
     Ok(())
 }
 
+/// Prepares a message to be send out
```

Comment:
This is kinda obvious from the method name, maybe you find a way to shortly describe also the difference to `prepare_msg_raw`?

Reply:
It internally uses prepare_msg_raw. Everything else it does is pretty much straight forward and preparation to use `prepare_msg_raw`. Would rather not add more documentation that also needs maintanace while the whole function is only 35 lines long and pretty easy to skim. 
I added this command in the effort of alex to finally remove the clippy rule for ignore missing docs, but we can also remove it again as it indeed doesn't tell too much.

Reply:
I see, in this case, it's fine either way I think.